deploy/helm/flightctl/templates/flightctl-db-deployment.yaml

```diff
@@ -40,10 +42,13 @@ spec:
           volumeMounts:
             - mountPath: /var/lib/pgsql/data
               name: flightctl-db
-{{ if .Values.db.resources }}
```

Comment:
why not define resources via helm values?

Reply:
Yes, this needs to continue to be configurable from the values. Some environments are restrictive in terms of limits or requests and won’t accept just anything

Reply:
Please leave it back and control the amount you need by setting it in values.db.resources